libs/blocks/notification/notification.js

```diff
@@ -0,0 +1,138 @@
+  const fontSizes = [...blockConfig[variant][size]];
+  if (el.classList.contains('s-button')) fontSizes.splice(3, 1, 'm');
+  return { fontSizes, options: { ...getOpts(el) } };
+}
+
+function decorateStaticLinks(el) {
+  const textLinks = el.querySelectorAll('a:not([class])');
+  textLinks.forEach((link) => { link.classList.add('static'); });
+}
+
+function decorateBorder(el, { borderBottom }) {
+  const border = createTag('div', { style: `background: ${borderBottom};`, class: 'border' });
+  el.appendChild(border);
+}
+
+function wrapCopy(foreground) {
+  const text = foreground.querySelector('.text');
+  if (!text) return;
+  const heading = text?.querySelector('h1, h2, h3, h4, h5, h6');
+  const icon = heading?.previousElementSibling;
+  const body = heading?.nextElementSibling?.classList.contains('action-area') ? '' : heading?.nextElementSibling;
+  const copy = createTag('div', { class: 'copy-wrap' }, [heading, body]);
+  text?.insertBefore(copy, icon?.nextSibling || text.children[0]);
+}
+
+function decorateClose(el) {
+  const btn = createTag('button', { 'aria-label': 'close', class: 'close' }, closeSvg);
+  btn.addEventListener('click', () => (el.style.display = 'none'));
+  el.appendChild(btn);
+}
+
+function decorateFlexible(el) {
+  const innards = [
+    el.querySelector('.background'),
+    el.querySelector('.foreground'),
+    el.querySelector('.close'),
+  ];
+  const inner = createTag('div', { class: 'flexible-inner' }, innards);
+  el.appendChild(inner);
+}
+
+function decorateLayout(el) {
+  const elems = el.querySelectorAll(':scope > div');
+  if (elems.length > 1) decorateBlockBg(el, elems[0]);
+  const foreground = elems[elems.length - 1];
+  foreground?.classList.add('foreground', 'container');
+  const text = foreground.querySelector('h1, h2, h3, h4, h5, h6, p')?.closest('div');
+  text?.classList.add('text');
+  const picture = text?.querySelector('p picture');
+  const iconArea = picture ? picture.closest('p') : null;
+  iconArea?.classList.add('icon-area');
+  const fgMedia = foreground.querySelector(':scope > div:not(.text) :is(img, video, a[href*=".mp4"])')?.closest('div');
+  const bgMedia = el.querySelector(':scope > div:not(.foreground) :is(img, video, a[href*=".mp4"])')?.closest('div');
+  const media = fgMedia ?? bgMedia;
+  el.classList.toggle('no-media', !media);
+  media?.classList.toggle('image', (media && !media.classList.contains('text')));
+  foreground?.classList.toggle('no-image', (!media && !iconArea));
+  if (el.matches(`:is(.${pill}, .${ribbon}):not(.no-closure)`)) decorateClose(el);
+  if (el.matches(`.${pill}.flexible`)) decorateFlexible(el);
+  return foreground;
+}
+
+export default function init(el) {
+  el.classList.add('con-block');
+  const { fontSizes, options } = getBlockData(el);
+  const blockText = decorateLayout(el);
+  decorateBlockText(blockText, fontSizes);
+  if (options.borderBottom) decorateBorder(el, options);
+  decorateTextOverrides(el);
+  decorateStaticLinks(el);
```

Comment:
I'd also get rid of this helper function, given it's a simply one liner
```suggestion
  el.querySelectorAll('a:not([class])').forEach((staticLink) => staticLink.classList.add('static'));
```